### DISCUSSION TITLE
foreign key queries for multiple rows from another table

### DISCUSSION THREAD

**Discussion by lmick002** [supabase/supabase#1100] [Questions] [answered]
Hey! 

So we're setting up supabase in our app and wanted to find out whats the best way to query all rows from another table that has a specific key value, for example

A Plumber might have multiple tools listed in a Tool table and we're querying the Plumbers row and also trying to include all of the Tools that have a plumber_id = 2 for example 

Thanks!

**Comment by silentworks** [comment, marked as answer]
If you have the relationship setup correctly then you can do something like

```js
supabase.from('plumbers')
  .select('*, tools(*)')
  .eq('id', 2)
  .single()
```

This should return the plumber and tools.

**Reply by lmick002** [reply]
thx @silentworks !
